lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -38,5 +41,30 @@ public PuppeteerLaunchTests()
 
             await browser.CloseAsync();
         }
+
+        [Fact]
+        public async Task NetworkRedirectsShouldReportSecurityDetails()
+        {
+            var options = TestConstants.DefaultBrowserOptions();
+            options.IgnoreHTTPSErrors = true;
```

Comment:
Let's try to use objects initializers when possible. I know it's only one prop, but just to keep the style.

Reply:
It's similar to Line 20 - https://github.com/kblok/puppeteer-sharp/pull/79/files/8a5d65d0f466541e0dececaa968dfd4b14022beb#diff-d88690bd1c588dd71b96f61f6c5c2b1fR20 and it's not an initialization, it's just a call to a static method so I'm not sure how we can use objects initializers

Reply:
That's true :) :+1: